### PR TITLE
add What to Cook?

### PR DESCRIPTION
**Pull request by kassner**
Thanks for taking the time to suggest an addition to awesome-selfhosted! 

To ensure your Pull Request is dealt with swiftly, please check the following (check the boxes `[x]`):
- [x] Submit one item per pull request. This eases reviewing and speeds up inclusion.
- [x] You have searched the repository for any relevant [issues](https://github.com/awesome-selfhosted/awesome-selfhosted-data/issues) or [PRs](https://github.com/awesome-selfhosted/awesome-selfhosted-data/pulls), including closed ones.
- [x] Any software you are adding is not already listed at any of [awesome-sysadmin](https://github.com/n1trux/awesome-sysadmin), [staticgen.com](https://www.staticgen.com/), [staticsitegenerators.net](https://staticsitegenerators.net/), [dbdb.io](https://dbdb.io/browse).
- [x] The file you are adding is formatted as described in [addition.md](https://github.com/awesome-selfhosted/awesome-selfhosted-data/blob/master/.github/ISSUE_TEMPLATE/addition.md).
- [x] `Demo` links should only be used for interactive demos, i.e. not video demonstrations. 
- [x] Comments and unused optional fields have been removed.
- [x] The file you are adding uses [kebab-case](https://en.wikipedia.org/wiki/Letter_case#Kebab_case) file naming, for example `my-awesome-software.yml`.
- [x] Values for `platform` are the main **server-side** requirements for the software. Don't include frameworks or specific dialects.
- [x] Any software project you are adding to the list is actively maintained.
- [x] Any software project you are adding was first released more than 4 months ago.
- [x] Any software project you are adding has working installation instructions.
- [x] You understand that your Pull Request will be merged ~1 week after approval, to allow for further comments if needed.

### CODE REVIEW

File: software/whattocook.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  - AGPL-3.0
+platforms:
+  - Java
+  - Docker
+tags:
+  - Recipe Management
+source_code_url: https://github.com/kassner/whattocook
+stargazers_count: 5
+updated_at: '2024-11-15'
+archived: false
```
Comment (Ki-er) [inline software/whattocook.yaml:14]:
```suggestion
```

**Review by Ki-er — CHANGES_REQUESTED** [review]
Quick change

File: software/whattocook.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+name: What To Cook?
+website_url: https://github.com/kassner/whattocook
+description: Click a button and get a recipe. You can exclude unavailable ingredients, but you can't just get another recipe.
+licenses:
+  - AGPL-3.0
+platforms:
+  - Java
```
Comment (kokomo123) [inline software/whattocook.yaml:7]:
```suggestion
```
Seeing as the only installation method is Docker, this isn't needed.

**Review by kokomo123 — APPROVED** [review]
LGTM, no issues here pending other thoughts from collaborators. Thanks for your contribution!

File: software/whattocook.yaml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+name: What To Cook?
+website_url: https://github.com/kassner/whattocook
+description: Click a button and get a recipe. You can exclude unavailable ingredients, but you can't just get another recipe.
```
Comment (nodiscc) [inline software/whattocook.yaml:3]:
```suggestion
description: Get a recipe to cook today, based on the ingredients you have at home.
```

**Review by nodiscc — CHANGES_REQUESTED** [review]

**Review by nodiscc — APPROVED** [review]

**Pull request merged**